NASA-PDS/pds-registry-common
# Dates

## 🗒️ Summary
Fixed parsing of "yyyyZ" dates. See https://github.com/NASA-PDS/harvest/issues/70

## ⚙️ Test Data and/or Report
- Test data is available in https://github.com/NASA-PDS/harvest/issues/70
- First build and install `pds-registry-common.jar` (this project).
- Next build Harvest. Make sure that Harvest uses the latest `pds-registry-common.jar` you just built/installed.
- Run Harvest to process a test file (you can use the file attached to https://github.com/NASA-PDS/harvest/issues/70 issue. That file has `pds:Time_Coordinates/pds:start_date_time` = "2015Z").
- Check `registry-docs.json` file in the Harvest output directory.
- `pds:Time_Coordinates/pds:start_date_time` should have "2015-01-01T00:00:00Z" value.

## ♻️ Related Issues
https://github.com/NASA-PDS/harvest/issues/70
